Packs/PAN-OS/Playbooks/PAN-OS_Commit_Configuration_v2.yml

```diff
@@ -0,0 +1,600 @@
+    continueonerror: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1140,
+          "y": 1610
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 3e46d493-9404-476d-8d95-68b8ad017a84
+    type: regular
+    task:
+      id: 3e46d493-9404-476d-8d95-68b8ad017a84
+      version: -1
+      name: Fix errors and commit manually
+      description: '-'
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "12"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 940,
+          "y": 735
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: 6e3293ac-bae0-4627-8b89-b25f67466c9f
+    type: regular
+    task:
+      id: 6e3293ac-bae0-4627-8b89-b25f67466c9f
+      version: -1
+      name: Fix errors and push manually
+      description: '-'
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1140,
+          "y": 1780
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "13_5_#default#": 0.1,
+      "13_6_Panorama": 0.73,
+      "14_12_yes": 0.24,
+      "15_17_#default#": 0.2,
+      "15_5_yes": 0.1,
+      "1_5_#default#": 0.17
+    },
+    "paper": {
+      "dimensions": {
+        "height": 2275,
+        "width": 1150,
+        "x": 370,
+        "y": -250
+      }
+    }
+  }
+inputs:
+- key: device-group
+  value: {}
+  required: false
+  description: ""
+  playbookInputQuery:
+outputs:
+- contextPath: Panorama.Commit.JobID
+  description: The job ID to commit.
+- contextPath: Panorama.Commit.Status
+  description: The commit status.
+- contextPath: Panorama.Commit.Description
+  description: The commit description from the the command input.
+- contextPath: Panorama.Push.DeviceGroup
+  description: The device group in which the policies were pushed.
+- contextPath: Panorama.Push.JobID
+  description: The job ID of the policies that were pushed.
+- contextPath: Panorama.Push.Status
+  description: The push status.
+- contextPath: Panorama.Push.Warnings
+  description: The push warnings.
+- contextPath: Panorama.Push.Errors
+  description: The push errors.
+- contextPath: Panorama.Push.Details
+  description: The job ID details.
+tests:
+- PAN-OS URL Filtering enrichment - Test
```

Comment:
the tests are related? old PB did not have those...

Reply:
I left one test that related